content/tutorials/define-and-provision-resources/index.md

```diff
@@ -1,56 +1,61 @@
 ---
-title_tag: "Defining and Provisioning Resources | Tutorials"
-meta_desc: Learn what a resource is, how resources work within Pulumi, and how to create, update, and delete resources in this tutorial.
 title: "Defining and Provisioning Resources"
-h1: "Defining and Provisioning Resources"
-meta_image: /images/docs/meta-images/docs-meta.png
-menu:
-  usingpulumi:
-    identifier: define-provision-resources
-    weight: 1
-tags:
-    - resources
-    - aws
-    - python
-    - nginx
-    - fundamentals
-    - tutorial
+title_tag: "Defining and Provisioning Resources"
```

Comment:
You'll need to add aliases to these to ensure traffic is redirected properly.

Reply:
just pushed the aliases